docs/source/api/queries.rst

```diff
@@ -529,7 +531,7 @@ Get Asset Info
 Purpose
 -------
 
-In order to know precision for given asset, and other related info in the future, such as a description of the asset, etc. user can send `GetAssetInfo` query.
+In order to know precision for given asset, and other information related to it (in the future - at the moment the option is not implemented), such as a description of the asset, etc. user can send `GetAssetInfo` query.
```

Comment:
Maybe just remove the part about the future information? It does not serve any purpose for this documentation. We can just say that asset info consists of precision.